docs/source/contrib.rst

```diff
@@ -69,3 +69,6 @@ Finally, pydoctor should never crash, no matter what code you feed it
 that universally applied, it seems).  Missing information is OK,
 crashing out is not.  This probably isn't as true as it should be at
 the moment.
+
+
+.. include:: <isonum.txt>
```

Comment:
this is ok. What I do is use `conf.py`  `rst_epilog` config to add these replacements or includes... so that you can use replacements without having to remember to include it in each page you use it.

Reply:
That sounds more convenient indeed. I'm still a Sphinx novice.